bindings/nodejs/examples/exchange/1-create-account.ts

```diff
@@ -30,8 +30,8 @@ async function run() {
                 '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
             );
         }
-        if (!process.env.MNEMONIC) {
-            throw new Error('.env MNEMONIC is undefined, see .env.example');
+        if (!process.env.NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1) {
```

Comment:
In the exchange guide it was extra made `MNEMONIC` for https://wiki.iota.org/next/iota-sdk/how-tos/exchange-guide/#1-generate-a-mnemonic
Would look weird to use `NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1` there

Reply:
ah, makes sense ... but then we need to change `.env.example` or have a separate for them, right?

Reply:
tbh ... I would call it `MNEMONIC` everywhere, it's simpler, and we already say in the docs of `.env.example` that those credentials should never be used for anything serious.

Reply:
The guide says to create one with the values

Reply:
yea i agree, its already weird that we have specific examples for exchange (even though theyre not so different).can at least have the code inside reused

Reply:
I would also rename everything to `MNEMONIC`

Reply:
Done